### DISCUSSION TITLE
C#对excel操作的开源库

### DISCUSSION THREAD

**Discussion by switchxiaomu** [BYJRK/DotNet-Discussions#116] [C# 技术交流]
大佬，NPOI是C#对excel操作的最佳选择吗？内存性能相关的损耗有优势吗？有没有教程推荐？
如果您能抽空给我回复或者建议，我将万分感谢！！！

**Comment by jlennonxzy** [comment]
EPPlus,我一直用的，只能读写excel，且是xlsx文件。

**Reply by Huyaohui** [reply]
EPPLUS,用起来代码不是很简洁，如果是想处里Excel文件，用python更合适

**Comment by CodingOctocat** [comment]
可以试试 [MiniExcel](https://github.com/mini-software/MiniExcel)。

**Reply by jlennonxzy** [reply]
我试试这个性能和资源占用如何，看起来不错，谢谢推荐。
